java/pom.xml

```diff
@@ -64,7 +64,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.16.1</version>
+            <version>3.21.12</version>
```

Comment:
Do you want to try to keep this in sync with the version of protoc used?

Reply:
@bdferris-v2 yes! this is in-sync, right? they should all be `[3.]21.12`

Reply:
Ah, I misunderstood the protobuf versioning scheme.